graph/api/src/main/java/org/jboss/windup/graph/model/comparator/FilePathComparator.java

```diff
@@ -30,39 +32,47 @@
     @Override
     public int compare(String o1, String o2)
     {
-        // if they are exactly the same, just short circuit everything
-        // and return 0
-        if (o1.equals(o2))
+        ExecutionStatistics.get().begin("FilePathComparator");
+        try
         {
-            return 0;
-        }
+            // if they are exactly the same, just short circuit everything
+            // and return 0
+            if (o1.equals(o2))
```

Comment:
I am just overthinking the issue.. String.equals(String) is not a big deal.. It is OK